2016-09/september-21.md

```diff
@@ -0,0 +1,49 @@
+## September 21 ([discuss](https://github.com/webpack/meeting-notes/pull/14))
+
+### Attendees
+
+* [Tobias Koppers](http://github.com/sokra)
+* [Sean Larkin](http://github.com/thelarkinn)
+* [Juho Vepsäläinen](http://github.com/bebraw)
+
+
+### Agenda
+
+#### Documentation Updates
+
+* Juho is going to start working on [antwar/antwar#103](https://github.com/antwarjs/antwar/issues/103) next week.
```

Comment:
I'll volunteer — I'm in documentation building mode right now anyways as I have 4 or 5 documentation sites I'm going to be putting together already in the next week or two. Would be easy to slip in another :-) I could have a PR w/ Gatsby ready by the end of the week.

My plan with Gatsby 1.0 and its theme/plugin support is to have a very robust "open source documentation site" theme with all the niceties that you could want. The nice thing about this is we'd be sharing the maintenance load for the Webpack site with a number of others.